testing/tests/headless.test.js

```diff
@@ -253,4 +253,39 @@ describe("Basic viewing of functional pages", () => {
     await expect(page).toMatch("<foo>: Une page de test");
     expect(page.url()).toBe(testURL("/fr/docs/Web/Foo"));
   });
+
+  it("clicking 'Sign in' should offer links to all identity providers", async () => {
```

Comment:
Is there a test missing about going to 'Sign in' when already logged in?

Reply:
Yes. But I'm not sure it's important enough to end-to-end test. Unless you have time to add a quick one. 

Also, what mayhaps would be interesting it to mock out the whole of Kuma there for those things. Then the headless tests could actually click the  "GitHub" button and be redirected and stuff. We're already doing similar mocking for the site-search. 

My only concern is that I've spent too much time on this already. Signing in is not important.

Reply:
I planned to get into Yari for some time, but my work keeps me spinning lately -.-
I'm sure, that end-to-end-test can be added afterwards.